agreements/employment.md

```diff
@@ -0,0 +1,167 @@
+<!--
+## Questions
+
+* We vs Us vs The Company
+* Are there other general duties?
+* Holiday practicalities
+    * Have a script which checks Forecast for work 28 days in the future and prompts people to book holiday if they want it
+    * Or an email with a link to confirm that they aren't taking any?
+    * Or just require 28 days notice and keep it manual?
+    * Or publish holiday somewhere and alert clients or DMs automatically to absences?
+    * Or push Breathe holiday into Forecast and then create some way for clients to see their project in Forecast?
+
+## Changes so far
+
+* Additional rights for dxw to make deductions from pay, permitting loan repayments, recovering expenses paid in error and recovering costs incurred due to negligence or dishonesty
+* Commission scheme removed, as it is not used
+* Probationary period increased to four months, with an option to extend for a further two
+* Notice period increased for staff who have been here for more than two years
+* Pension scheme added
+* Employee right to claim expenses added
+* Parental leave added
```

Comment:
Sorry, but I can't find any reference to is here: https://github.com/dxw/playbook/blob/master/playbook.md